extra-x11/nvidia-prime/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=nvidia-prime
+PKGSEC=x11
+PKGDES="NVIDIA Prime Render Offload configuration and utilities"
+PKGDEP="libglvnd mesa vulkan"
```

Comment:
I thought you mentioned that it depends on `nvidia-utils` or something?

Reply:
Yes. It depends on `nvidia-utils` which provides `nvidia-libgl, opengl-driver, vulkan-driver` in Arch Linux.

Reply:
Okay, in this case, please add `nvidia` to `$PKGDEP`.

Reply:
6521dd